libs/common-types/src/lib.rs

```diff
@@ -63,9 +105,28 @@ bitflags::bitflags! {
 		const LIQUIDITY_ADMIN = 0b00001000;
 		const MEMBER_LIST_ADMIN = 0b00010000;
 		const RISK_ADMIN = 0b00100000;
+		const PERMISSIONED_ASSET_MANAGER = 0b01000000;
+		const PERMISSIONED_ASSET_ISSUER = 0b10000000;
```

Comment:
We can think about extending the `PermissionRoles` as 

```rust
struct PermissionRoles {
    pool_admin: PoolAdminRoles,
    curreny_admin: CurrencyAdminRoles,
    permissioned_asset_holder: PermissionedCurrencyHolders<Now, MinDelay, Moment>,
    tranche_investor: TrancheInvestors<Now, MinDelay, TrancheId, Moment>,
```

if we wanna split the admin roles further. But just an idea.

Reply:
Hm yes, I think that's a good idea! Will explore this.